.github/pages/index.md

```diff
@@ -1,13 +1,11 @@
 ---
-title:  "Editor Documentation"
+title:  "HowTo for Admins"
```

Comment:
Could we rename this to **HowTo for Content Editors** - in our context _admin_ would (more often than not) refer to `wp-admin` or, _the Dashboard_.